actionview/test/ujs/public/vendor/jquery-2.2.0.js

```diff
@@ -6852,7 +6852,7 @@ jQuery.speed = function( speed, easing, fn ) {
 		opt.queue = "fx";
 	}
 
-	// Queueing
+	// Queuing
```

Comment:
This is a vendor file we didn't create, I don't think we should change this.

Reply:
My mistake, good catch!